tests/nnstreamer_filter_onnxruntime/runTest.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+##
+## SPDX-License-Identifier: LGPL-2.1-only
```

Comment:
I am not sure but you need to check the license for the script.

Reply:
Thanks. updated it.